xdebug_stack.c

```diff
@@ -800,7 +800,11 @@ void xdebug_error_cb(int type, const char *error_filename, const uint error_line
 			break;
 	}
 
+#if PHP_VERSION_ID >= 70200
+	if (PG(track_errors) && EG(active)) {
```

Comment:
EG(active) has been available for quite awhile.

You can probably get away with *just* using this version and not bothering with the valid_symbol_table check at all.